backend/functions/src/email/templates.ts

```diff
@@ -0,0 +1,257 @@
+const matched = () => {
+  return `Subject: Study Partners!
+  <div> 
+    Dear students,  
+  </div>
+    <br> 
+    <br> 
```

Comment:
I'm speaking for this whole file, and I wanted to address this before, but is there a reason that everything is hardcoded? I feel like adopting a strategy with how the survey questions are made with JSON files as opposed to hardcoding (data-driven 😩) would be a way to go, especially with customization.

Reply:
@mluo24 could do that, but then we have to find/replace magic values to substitute for variables instead of using string template literals. Good point tho